docs/flex.md

```diff
@@ -35,14 +35,14 @@ The following functions are defined:
 * `osm2pgsql.define_table()`: Define a table. This is the more flexible
   function behind all the other `define_*_table()` functions. It gives you
   more control than the more convenient other functions.
-* `osm2pgsql.mark_way(id)`: Mark the OSM way with the specified id. This way
-  will be processed (again) in stage 2.
 
 You are expected to define one or more of the following functions:
 
-* `osm2pgsql.process_node()`: Called for each node.
-* `osm2pgsql.process_way()`: Called for each way.
-* `osm2pgsql.process_relation()`: Called for each relation.
+* `osm2pgsql.process_node()`: Called for each new or changed node.
+* `osm2pgsql.process_way()`: Called for each new or changed way.
+* `osm2pgsql.process_relation()`: Called for each new or changed relation.
+* `osm2pgsql.mark_relation_members()`: Called for each deleted or added
```

Comment:
I'm still not quite happy with the name. The function doesn't mark anything, it returns members to be reprocessed.

Reply:
Now called `select_relation_members()`. Not perfect but maybe a bit better.